avogadro/qtplugins/noncovalent/noncovalent.cpp

```diff
@@ -80,6 +88,27 @@ static enum InteractionTypes getInteractionType(const Molecule &molecule, Index
         }
       }
       break;
+    case 9: case 17: case 35: case 53: // halogen bond
+      for (const Bond *b : molecule.bonds(i)) {
+        Index j = (b->atom1().index() == i ? b->atom2() : b->atom1()).index();
+        unsigned char jnum = molecule.atomicNumber(j);
+        switch (jnum) {
+          case 6: case 7: case 8: case 9:
+          case 16: case 17: case 35: case 53: // F, O, N, Cl, Br, C, I, S
+            return HALOGEN_BOND;
+        }
+      }
+      break;
+    case 8: case 16: // chalcogen bond
```

Comment:
8, 16, 34 for sure. Some people may want 52 as well... probably safe to add it, since Te-Te interactions don't happen very often.